### PR TITLE
LBSD-824 highlights in timeline

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

**Comment by vied12** [comment]
:+1: and e2e tests would be useful here

File: client/app/scripts/liveblog-edit/directives.js [hunk 1]
```diff
@@ -253,6 +256,26 @@ define([
                                     notify.error(gettext('Something went wrong. Please try again later'));
                                 });
                             },
+                            highlightPost: function(post) {
+                                scope.clearReorder();
+                                if (post.highlight == false) {
```
Comment (nistormihai) [inline client/app/scripts/liveblog-edit/directives.js]: please remove this is is not needed.
you can just use two messages base on the highlight.
ex: notify.info( post.highlight? gettext('Post was highlighted') ...

File: client/app/scripts/liveblog-edit/directives.js [hunk 2]
```diff
@@ -253,6 +256,26 @@ define([
                                     notify.error(gettext('Something went wrong. Please try again later'));
                                 });
                             },
+                            highlightPost: function(post) {
+                                scope.clearReorder();
```
Comment (nistormihai) [inline client/app/scripts/liveblog-edit/directives.js]: this is not needed, reorder isn't affected by highlight.

**Comment by nistormihai** [comment]
LGTM great work :+1:

**Pull request merged**